lib/column/decimal.go

```diff
@@ -92,6 +92,11 @@ func (col *Decimal) Row(i int, ptr bool) interface{} {
 
 func (col *Decimal) ScanRow(dest interface{}, row int) error {
 	switch d := dest.(type) {
+	case *float64:
```

Comment:
This potentially results in precision loss e.g. float64 can't represent numbers exactly like decimal. This is against a philosophy we apply in v2 that I think v1 got wrong i.e. it allowed precision loss. The user should accept this in their code explicitly vs the driver doing it for them.

Reply:
Ok, what about IP and UUID?

Reply:
Removed